rcsb/utils/dictionary/DictMethodCommonUtils.py

```diff
@@ -4452,9 +4468,12 @@ def getCompModelLocalScores(self, dataContainer):
         try:
             if dataContainer.exists("ma_qa_metric_local"):
                 tObj = dataContainer.getObj("ma_qa_metric_local")
+                repModelId = self.getRepresentativeModelId(dataContainer)
```

Comment:
The issue of obtaining representative models for CSMs exists here as well and needs to be addressed when the code is consolidated.